templates/index.html

```diff
@@ -89,14 +104,10 @@ <h3 class="p-heading--2">Infrastructure, <br class="u-hide--medium u-hide--small
         <hr class="p-rule is-dark">
         <div class="row">
           <div class="col-6">
-            <h3 class="p-heading--2">Hybrid, multi-cloud <br class="u-hide--medium u-hide--small">experience</h3>
-            <p>Run your workloads anywhere with cloud-agnostic Kubernetes and tooling that helps you deploy, integrate and operate software across clouds.</p>
+            <h3 class="p-heading--2">Zero-ops applications <br class="u-hide--medium u-hide--small">on any infrastructure</h3>
```

Comment:
I think it should read "... on any cloud" instead of "... on any infrastructure"

Reply:
the text is usually verified by UX, @juanruitina would you agree with Johann's correction?

Reply:
Gloria specifically changed it to that language, so keep as is please